src/vl-mapactions-snap-interaction.js

```diff
@@ -1,12 +1,9 @@
 import Snap from 'ol/interaction/Snap';
 
 export class VlSnapInteraction extends Snap {
-  constructor(layer) {
-    const source = layer ? layer.getSource() : null;
-    const snapOptions = {
-      source: source,
-      pixelTolerance: 7,
-    };
+  constructor(options) {
+    const snapOptions = Object.assign({}, options);
+    snapOptions.pixelTolerance = options.pixelTolerance || 7;
     super(snapOptions);
     this.snapOptions = snapOptions;
```

Comment:
De `this.snapOptions` wordt alleen gebruikt in de testen, maar is het niet beter om in de testen de super configuratie te controleren?

Reply:
Dat stond er al zo, maar ik zal eens kijken of dat lukt.

Reply:
Geen mogelijkheid om publiek op te vragen wat er in die snap interactie zit maar ik zal de private fields gebruiken in de unit tests.